include/envoy/server/instance.h

```diff
@@ -224,6 +224,12 @@ class Instance {
    *         messages.
    */
   virtual ProtobufMessage::ValidationContext& messageValidationContext() PURE;
+
+  /**
+   * @param is_dynamic true if the validation visitor is for dynamic config
```

Comment:
Can we name the parameter something more clear for what this does? It's not super clear to me what is different about the returned context so perhaps more comments?